src/main/java/org/mule/extension/http/api/listener/intercepting/cors/CorsInterceptorWrapper.java

```diff
@@ -27,4 +29,19 @@
   public HttpListenerInterceptor getInterceptor() {
     return corsInterceptor;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o)
+      return true;
+    if (o == null || getClass() != o.getClass())
+      return false;
+    CorsInterceptorWrapper that = (CorsInterceptorWrapper) o;
```

Comment:
We need to work with the API Gateway team which maintains this implementation so that they provide a proper hash and equals here, otherwise we will have issues all the same.

Reply:
@balbifm what do you think?

Reply:
@mtqp what do you think?

Reply:
We have released Cors Kernel 1.1.2, the HTTP Connector should point to that version now.